code/kit_version.R

```diff
@@ -0,0 +1,107 @@
+# SET UP
+setwd("C:/Users/kitsc/KIT_code_testing/data")
+library(haven)
+library(tidyverse)
+library(plm)
+library(dplyr)
+
+# DEFINITIONS
+
+##	seed
+	seed=51
+	set.seed(seed)
+##	number of repetitions
+	reps=5000 
+##	sd of error term
+	sd = 3000
+##	coefficients on temperature term (hypothetical linear response)
+	beta = 0.25
+##	coefficients on long-run temperature interaction term
+	gamma = 0.02
+##	constant term
+	alpha = 12000
+##	difference in pixel temperature and polynomial means
+	pixel_meantemp_diff = 20
+
+# REAL COUNTRY-LEVEL TEMP DATA
+
+##	importing
+	real_temps <- read_dta("Europe_temps.dta")
+	d <- real_temps
+##	create country ID
+	d$group <- d %>% group_indices(country) 
+##	create count of countries
+	country_count = length(unique(d$group))
+##	create mean-temp-by-country variable & deviations variable
+	d = d %>%
+	  group_by(country) %>%
+	  mutate(avgtemp1 = mean(temp1),
+	  		deviation = temp1 - avgtemp1) %>%
+	  ungroup()
+
+# FAKE DATA
+
+##	Generate positively and negatively correlated pixel temperature
+	d = d %>%
```

Comment:
A slightly cleaner way to write this is to use the `%<>%` operator, which is the same as doing `x = x %>% ...`
```suggestion
	d %<>%
```